src/ecdsa/keys.py

```diff
@@ -649,6 +656,15 @@ def __init__(self, _error__please_use_generate=None):
         self.baselen = None
         self.verifying_key = None
         self.privkey = None
+        
+    def __eq__(self, other):
+        """Return True if the points are identical, False otherwise."""
```

Comment:
it's talking about points but is also checking `default_hashfunc`...

Reply:
also, I wonder if taking default_hashfunc into account is correct at all

Reply:
yeah, hash here is more about implementation than instance of the key